packages/tooltip/stories.tsx

```diff
@@ -55,13 +54,12 @@ storiesOf('Tooltip Container/useTooltip', module)
       const triggerRef = useRef(null);
 
       const { isVisible, getTooltipProps, getTriggerProps } = useTooltip({
-        tooltipRef,
         isVisible: boolean('isVisible', false),
         delayMilliseconds: number('Tooltip delay', 500)
       });
       const { style } = usePopper({ referenceRef: triggerRef, popperRef: tooltipRef });
 
-      const styles = {
+      const styles: CSSStyleDeclaration | React.CSSProperties = {
```

Comment:
Explicit typing is used here because it provides the benefit of TypeScript editor support. For example, TypeScript shows an error when an invalid CSS like `color: false` is used 🚀. Without the explicit type, the value of `color` would be inferred as a `boolean` and no error is shown — which isn't helpful.